docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

```diff
@@ -382,6 +382,195 @@ they are listed for each job.
 |https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v3_windows_rare_user_type10_remote_login.json[image:images/link.svg[A link icon]]
 |https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/datafeed_v3_windows_rare_user_type10_remote_login.json[image:images/link.svg[A link icon]]
 
+|===
+// end::security-windows-jobs[]
+
+[discrete]
+[[security-integrations-jobs]]
+== Security: Elastic Integrations
+
+https://docs.elastic.co/integrations[Elastic Integrations] are a streamlined way to add Elastic assets to your environment, such as data ingestion, transforms, and in this case, machine learning capabilities for Security.
+
+The following Integrations are for the Security UEBA (user and entity behavioral analytics), which can help detect and alert when there is related suspicious activity in your environment.
+
+* https://docs.elastic.co/integrations/ded[Data Exfiltration Detection]
+* https://docs.elastic.co/integrations/dga[Domain Generation Algorithm Detection]
+* https://docs.elastic.co/integrations/lmd[Lateral Movement Detection]
+* https://docs.elastic.co/integrations/problemchild[Living off the Land Attack Detection]
+
+// tag::security-integrations-jobs[]
+
+*Domain Generation Algorithm (DGA) Detection (Integration)*
+
+ML solution package to detect domain generation algorithm (DGA) activity in your network data. Requires a Platinum subscription.
+
+To download, see the https://docs.elastic.co/integrations/dga[documentation].
+
+|===
+|Name |Description |Job |Datafeed
+
+|dga_high_sum_probability
+|Detect domain generation algorithm (DGA) activity in your network data.
+|https://github.com/elastic/integrations/blob/{branch}/packages/dga/kibana/ml_module/dga-ml.json#L23[image:images/link.svg[A link icon]]
+|https://github.com/elastic/integrations/blob/{branch}/packages/dga/kibana/ml_module/dga-ml.json#L58[image:images/link.svg[A link icon]]
```

Comment:
Why these links aren't working?

Reply:
Hi, thanks for looking, they are working on the [preview](https://stack-docs_2496.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-ml-jobs.html#security-integrations-jobs) linked in the PR body